.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
what is this trickery?

Reply:
It's so `chruby` will auto switch when you navigate to the project in terminal.